website/docs/developer/deployment-guide/recovery-workflow.mdx

```diff
@@ -34,3 +34,100 @@ directly or by checking the status on the Cumulus Dashboard.
     zoomOutPic={useBaseUrl('img/zoom-out.svg')}
     resetPic={useBaseUrl('img/zoom-pan-reset.svg')}
 />
+
+See the recovery workflow [stepfunction modules](https://github.com/nasa/cumulus-orca/tree/master/modules/workflows/OrcaRecoveryWorkflow) 
+for additional details on recovering files belonging to a granule.
+
+## Recovery workflow parameters
+
+The following are the parameters needed for recovery workflow:
+
+- buckets- AWS S3 bucket mapping used for Cumulus and ORCA configuration. Contains the following properties:
+  - name (Required)- Name of the S3 bucket.
+  - type (Optional)- the type of bucket - i.e. internal, public, private, protected. 
+
+  It can be set up using the following configuration.
+  ```json
+        "config": {
+          "buckets.$": "$.meta.buckets"
+        }
+    ```
+    Example:
+
+    ```json
+    "buckets": {
+            "protected": {"name": "sndbx-cumulus-protected", "type": "protected"},
+            "internal": {"name": "sndbx-cumulus-internal", "type": "internal"},
+            "private": {"name": "sndbx-cumulus-private", "type": "private"},
+            "public": {"name": "sndbx-cumulus-public", "type": "public"}
+        }
+    ```
```

Comment:
Anybody know why we're requiring this to be passed into the step function, and not just using the value from tfvars?